src/docs/sdk/event-payloads/contexts.mdx

```diff
@@ -149,6 +149,54 @@ The `type` and default key is `"device"`.
 
 : _Optional_. The language of the device. For example, `en-US`.
 
+`processor_count`
+
+: _Optional_. Number of "logical processors". For example, `8`.
+
+`cpu_description`
+
+: _Optional_. CPU description. For example, `Intel(R) Core(TM)2 Quad CPU Q6600 @ 2.40GHz`.
+
```

Comment:
I'm just curious, why do you need this when you already report the phone model?

Reply:
I personally found that interesting. Possibly even more to those building games that run on a variety of devices like desktop, console and mobile.